packages/react/src/transactions/adapters/PendingTransactionGateway/schema/publications.ts

```diff
@@ -15,7 +15,7 @@ import { Erc20AmountSchema } from './common';
 const NftAttributeSchema = z.union([
   z.object({
     displayType: z.literal(NftAttributeDisplayType.Date),
-    value: z.date(),
+    value: z.coerce.date(),
```

Comment:
Can you please explain to me what the changes does?

Reply:
from zod's docs: https://zod.dev/?id=dates

Coercion to Date

Since [zod 3.20](https://github.com/colinhacks/zod/releases/tag/v3.20), use [z.coerce.date()](https://zod.dev/?id=coercion-for-primitives) to pass the input through new Date(input).

```
const dateSchema = z.coerce.date();
type DateSchema = z.infer<typeof dateSchema>;
// type DateSchema = Date

/* valid dates */
console.log(dateSchema.safeParse("2023-01-10T00:00:00.000Z").success); // true
console.log(dateSchema.safeParse("2023-01-10").success); // true
console.log(dateSchema.safeParse("1/10/23").success); // true
console.log(dateSchema.safeParse(new Date("1/10/23")).success); // true

/* invalid dates */
console.log(dateSchema.safeParse("2023-13-10").success); // false
console.log(dateSchema.safeParse("0000-00-00").success); // false
```